bucket/anyburn.json

```
@@ -0,0 +1,36 @@
+{
+    "version": "4.5",
+    "description": "Free Solutions for CD / DVD / Blu-ray burning and imaging.",
+    "homepage": "http://www.anyburn.com/",
+    "license": {
+        "identifier": "Freeware",
+        "url": "http://www.anyburn.com/tutorials/eula.htm"
+    },
+    "url": "http://www.anyburn.com/anyburn.zip",
+    "hash": "d1d0528e75557fbba709849d8f7d4d5dd1cbca0535e8cb5176a688e6d3c15055",
+    "architecture": {
+        "32bit": {
+            "extract_dir": "AnyBurn(32-bit)"
+        },
+        "64bit": {
+            "extract_dir": "AnyBurn(64-bit)"
+        }
+    },
+    "bin": [
+        "AnyBurn.exe",
+        "abcmd.exe"
+    ],
+    "shortcuts": [
+        [
+            "AnyBurn.exe",
+            "AnyBurn"
+        ]
+    ],
+    "checkver": {
+        "url": "http://www.anyburn.com/download.php",
+        "regex": "<b>AnyBurn v([\\d]+[\\.][\\d]+)"
+    },
+    "autoupdate": {
+        "url": "http://www.anyburn.com/anyburn.zip"
```

Add hash extraction from download page:

```suggestion
        "url": "http://www.anyburn.com/anyburn.zip",
        "hash": {
            "url": "http://www.anyburn.com/download.php",
            "regex": "MD5:\\s+$md5"
        }
```